src/components/Greeting.js

```diff
@@ -0,0 +1,15 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchMessage } from '../redux/features/greetingSlice';
+
+const Greeting = () => {
+  const dispatch = useDispatch();
+  const greeting = useSelector((state) => state.greeting.text);
+  useEffect(() => {
+    // console.log('useEffect is running');
```

Comment:
- [ ] Your code is well-written and functional. However, consider removing commented-out sections to enhance clarity and maintainability, ensuring a cleaner and more streamlined codebase. Great effort overall!